lookout/core/helpers/analyzer_context_manager.py

```diff
@@ -0,0 +1,97 @@
+from argparse import Namespace
+import subprocess
+from typing import Optional, Type
+
+from lookout.core.analyzer import Analyzer
+from lookout.core.cmdline import create_model_repo_from_args
+from lookout.core.data_requests import DataService
+from lookout.core.event_listener import EventListener
+from lookout.core.helpers.server import check_port_free, find_port, LookoutSDK
+from lookout.core.manager import AnalyzerManager
+
+
+class AnalyzerContextManager:
+    """Context manager for launching analyzer."""
+
+    def __init__(self, analyzer: Type[Analyzer], db: str, fs: str,
+                 init: bool = True, data_request_address: str = "localhost:10301"):
+        """
+        Initialization.
+
+        :param db: path to an SQLite database with model metadata.
+        :param fs: location where to store the trained model. (right?)
```

Comment:
Hehe, I actually asked this "right?" because I was not sure

Reply:
oops. :)